src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoresAndReports.tsx

```diff
@@ -67,6 +67,7 @@ export const ScoresAndReports = () => {
     name: reportsName,
     keyName: REACT_HOOK_FORM_KEY_NAME,
   });
+  const reportsSize = reports?.length ?? 0;
```

Comment:
```suggestion
  const reportsSize = reports?.length ?? null;
```

Reply:
No need, since we use this value later in comparison